source/kraken/fill_disruption_from_database.h

```diff
@@ -173,9 +173,9 @@ namespace navitia {
         template<typename T>
         void fill_pt_object(T const_it) {
             auto ptobject = impact->add_informed_entities();
-            FILL_REQUIRED(ptobject, updated_at, uint64_t)
-            FILL_NULLABLE(ptobject, created_at, uint64_t)
-            FILL_REQUIRED(ptobject, uri, std::string)
+            FILL_NULLABLE(ptobject, updated_at, uint64_t)
+            FILL_REQUIRED(ptobject, created_at, uint64_t)
+            FILL_NULLABLE(ptobject, uri, std::string)
```

Comment:
but it is nullable in the database...
I have opened an issue in chaos: http://jira.canaltp.fr/browse/IS-1031